bindings/pyroot/pythonizations/src/PyROOTModule.cxx

```diff
@@ -60,6 +61,69 @@ PyObject *RegisterExecutorAlias(PyObject * /*self*/, PyObject *args)
    Py_RETURN_NONE;
 }
 
+namespace Internal {
```

Comment:
```suggestion
```
Just a nitpick: but, `.cxx` sources are always internal anyway so `Internal`/`Detail` namespaces have no meaning IMHO

Reply:
The equivalent for a source file is to use the anonymous namespace.  Note that both solution (anonymous namespace or Internal namespace) had the advantage of hiding (mangling) the `PyObjRefCounter` symbols from the linker.

Reply:
Yes I have thought about that as well with the linker, it's a good point. But it's already in the `PyROOT` namespace, which I double will collide with anything (`PyROOT::Internal` is not reducing the probability that much further)